frontend/website/docs/index.md

```diff
@@ -11,3 +11,17 @@ This breakthrough is made possible due to zk-SNARKs - a type of succinct cryptog
 Trying out Coda is simple - head over to the [Getting Started page](/docs/getting-started/) to install the Coda client and join the network.
 
 You can also sign up for the public testnet updates [here](https://bit.ly/TestnetForm).
+
+<span id="testnet-points" />
+## Testnet Points[\*](#disclaimer)
```

Comment:
I don't think this parses correctly

Reply:
lol it parses but looks a bit funky 
<img width="280" alt="Screen Shot 2019-07-24 at 6 22 03 PM" src="https://user-images.githubusercontent.com/1709621/61838846-0ebe4000-ae40-11e9-87ce-54ebe71eb1d3.png">

I don't think we need the link.
```suggestion
## Testnet Points*
```